pkgs/by-name/wa/wasabiwallet/package.nix

```
@@ -58,13 +58,13 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/opt/${pname} $out/bin $out/share/applications
-    cp -Rv . $out/opt/${pname}
+    cp -Rv ./runner/work/WalletWasabi/WalletWasabi/build/linux-x64/* $out/opt/${pname}
```

shouldn't this include "home" element?

```suggestion
    cp -Rv ./home/runner/work/WalletWasabi/WalletWasabi/build/linux-x64/* $out/opt/${pname}
```

also this feels like an upstream packaging error ... can you include a comment that says this is a upstream packaging error and that the path will probably be fixed in the upcoming release?

---

Yes, it is and error because that's the path where the github action builds the release and that is not okay. Unfortunately the binaries are there and there is no alternative other than copying from there.

Adding './home` won't work because the files are not there.

---

`tar tfz file.tar.gz` tells me the files are in `home/runner/work/...` and not `runner/work/...`

---

Yes, many many times but my experience is very limited. This is how I build it: `nix-build -E 'with import <nixpkgs> {}; callPackage ./package.nix {}'`. After that I run `./result/bin/wasabiwallet-desktop` and it works. The other executables in `./result/bin/` also work well. 

If I apply your suggestion the derivation doesn't build.

---

yes, i realized my own mistake misunderstanding how tar works. sorry